### PR TITLE
SPR: Mention the support for K3s + traefik/nginx (CAPS-149)

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: adoc/Private-Registry-Installation.adoc [hunk 1]
```diff
@@ -316,6 +316,9 @@ internalTLS:
 ====
 Internal TLS support does not yet cover the internal database and Redis services.
 ====
++
+If {spr} is deployed in k3s, note that unmodified traefik (the default ingress controller bundled with k3s) will not work with automatically generated certificates.
```
Comment (flaviodsr) [inline adoc/Private-Registry-Installation.adoc:320]: As there are many certificate configurations maybe its worthy mentioning that it won't work when using internalTLS with automatically generated certificates?

File: adoc/Private-Registry-Installation.adoc [hunk 2]
```diff
@@ -316,6 +316,9 @@ internalTLS:
...
+For such setup, It is necessary to add `insecureSkipVerify = true` option for traefik. Learn how to modify traefik settings in link:https://rancher.com/docs/k3s/latest/en/helm/#customizing-packaged-components-with-helmchartconfig[upstream documentation].
```
Comment (flaviodsr) [inline adoc/Private-Registry-Installation.adoc:321]: Does this needs to be on another paragraph? I would put this together with the previous paragraph.

Mabye `...it is necessary to configure traefik  to not verify the backend SSL certificate (insecureSkipVerify = true)...`

**Review by flaviodsr — CHANGES_REQUESTED** [review]

File: adoc/Private-Registry-Installation.adoc (hunk 1 above)
Comment (jsuchome) [reply adoc/Private-Registry-Installation.adoc:320]: Well, this text is in the section that already describes internalTLS, so I think it's clear from the context...

Comment (flaviodsr) [reply adoc/Private-Registry-Installation.adoc]: indeed! 👍

**Review by flaviodsr — APPROVED** [review]

**Review by stefannica — APPROVED** [review]

**Pull request merged**